knockout.popupTemplate.js

```diff
@@ -238,11 +238,18 @@ Source code found at https://github.com/One-com/knockout-popupTemplate
         callMeMaybe(done);
     };
 
-    Popup.prototype.reposition = function () {
+    Popup.prototype.reposition = function (e) {
         if (!this.$popupHolder) { return; }
 
         this.bestPosition(this.getBestPosition());
         var boundingRect = this.$popupHolder[0].getBoundingClientRect();
```

Comment:
Could these lines not be pushed down after the scroll check? The `bestPosition` obverable is being used in the calculations in the method `calculateInitialPosition` but there's no reason to update it if we don't actually set any position, is there?